modules/oadp-installing-dpa.adoc

```diff
@@ -181,28 +181,29 @@ spec:
         provider: {provider}
         default: true
         credential:
-          key: cloud
-          name: {credentials} <5>
+          key: cloud <5>
+          name: {credentials} <6>
         objectStorage:
-          bucket: <bucket_name> <6>
-          prefix: <prefix> <7>
-  snapshotLocations: <8>
+          bucket: <bucket_name> <7>
+          prefix: <prefix> <8>
+  snapshotLocations: <9>
     - velero:
         provider: {provider}
         default: true
         config:
           project: <project>
-          snapshotLocation: us-west1 <9>
+          snapshotLocation: us-west1 <10>
 ----
 <1> The `openshift` plugin is mandatory.
 <2> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
 <3> Set this value to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that Restic pods run on each working node. In OADP version 1.2 and later, you can configure Restic for backups by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR. In OADP version 1.1, add `spec.defaultVolumesToRestic: true` to the `Backup` CR.
 <4> Specify on which nodes Restic is available. By default, Restic runs on all nodes.
-<5> If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
-<6> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
-<7> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
-<8> Specify a snapshot location, unless you use CSI snapshots or Restic to back up PVs.
-<9> The snapshot location must be in the same region as the PVs.
+<5> Secret key that contain credentials. For Google workload identity federation cloud authentication use `service_account.json`.
```

Comment:
Mention that it is because the key 'service_account.json' was generated in the secret in the GCP-WIF section, the user can change this key to any value he wants in the secret manifest and apply the secret and sync it here in the DPA.

Reply:
@sbahar619 
Don't pay attention to the oadp-installing-dpa section. It will be replaced by two new sections - oadp-installing-dpa-1-2-and-earlier.adoc modules/oadp-installing-dpa-1-3.adoc in this PR: https://github.com/openshift/openshift-docs/pull/66683